File: sample/src/main/java/io/erva/sample/base/cell/BetaCell.java
package io.erva.sample.base.cell;

import android.view.View;
import android.widget.TextView;

import butterknife.BindView;
import io.erva.celladapter.Cell;
import io.erva.celladapter.Layout;
import io.erva.sample.BaseCell;
import io.erva.sample.R;
import io.erva.sample.base.model.BetaModel;

@Layout(R.layout.item_base_beta)
public class BetaCell extends BaseCell<BetaModel, BetaCell.Listener> {

    @BindView(R.id.tv_beta) TextView textView;

    public BetaCell(View view) {
        super(view);
    }

    @Override
    protected void bindView() {
        textView.setText(getItem().getBeta());
    }

    public interface Listener extends Cell.Listener<BetaModel> {

    }
}

File: sample-v7-kotlin/src/main/kotlin/io/erva/sample/base/cell/BetaCell.kt
package io.erva.sample.base.cell

import android.view.View
import io.erva.celladapter.Layout
import io.erva.celladapter.v7.Cell
import io.erva.sample.R
import io.erva.sample.base.model.BetaModel
import kotlinx.android.synthetic.main.item_base_beta.view.*

@Layout(R.layout.item_base_beta)
class BetaCell(view: View) : Cell<BetaModel, BetaCell.Listener>(view) {

    override fun bindView() {
        view.tv_beta.text = item().beta
    }

    interface Listener : Cell.Listener<BetaModel>
}